### ISSUE TITLE
Is there a rule for which extensions are included?

### ISSUE THREAD

**Issue by Nemo64** [Chialab/docker-php#15]
I work with typo3 which requires the extensions bcmath and soap.
BCMath is secretly also required by a swiftmailers NTLMAuthenticator. (just as a more common usage example)
It is fairly easy to install them afterwards but still a step slower than if they were installed already.
Is there a rule for which extensions are included?

**Comment by fquffio** [comment]
No, there isn't.

We created this image to meet our own needs: we work mainly with [CakePHP](https://cakephp.org), so we installed the extensions required by the framework, plus a small set of extensions we commonly use. I honestly see no drawbacks of having an image with a few more extensions than strictly needed: some prefer Docker images to have a very small footprint, and to be as "slim" as possible, but in that case the only real solution IMHO is to build an image from scratch.

I don't think that many people rely on this image right now, so we can basically do what we want. 🙃  BCmath is surely a 👍 . Not really excited about SOAP as it sounds a little bit "old school" to me, but it's surely a matter of irrational taste. Also, the first [user contributed note here](http://php.net/manual/en/soap.installation.php) doesn't look promising. 😂
